cli/githubutil.py

```diff
@@ -146,18 +150,110 @@ def list_draft_releases(
     github_cfg_name: str,
     github_repository_owner: str,
     github_repository_name: str,
+    greatest_release_version: CliHint(
```

Comment:
would it not suffice to determine the greatest released version (a functionality which we already have ready-to-use)?